yml/dp02_patches.yaml

```diff
@@ -0,0 +1,58 @@
+---
+name: dp02_dc2_images
+"@id": "#dp02_dc2_images"
+description: Information about image data products in DP0.2 processing of DC2
+tables:
+- name: lsst_skymap
+  "@id": "#lsst_skymap"
+  description: Static information about the tract-and-patch skymap used for coadds
+  columns:
+  - name: lsst_tract
+    "@id": "#lsst_skymap.lsst_tract"
+    description: "ID number of the top level, 'tract', of the standard LSST skymap"
```

Comment:
There's no standard LSST skymap; this data is particular to the DC2 skymap.

Reply:
I don't understand that - I thought the tracts were defined once for all time?  Does, say, HSC use a different tiling?

Reply:
Yes, HSC has a different skymap (hsc_rings_v1). Each skymap can have an arbitrarily different mapping of tracts to position on the sky.